source/docs/advanced/payments/payment-workflows.md

```diff
@@ -0,0 +1,67 @@
+---
+id: payment-workflows
+title: Understanding payment workflows
+---
+
+Front-Commerce provides different hooks allowing to use the payment method of your choice. This section explains the high-level payment workflows supported.
+
+Front-Commerce has a concept of "payment workflow" to model a payment scenario from a customer standpoint. This is a core concept to understand for anyone wanting to create a really custom checkout or implement a new payment method. It is important that developers understand them so they could choose a payment method knowingly.
+
+So far, 3 workflows are supported and are explained below:
+- Direct Order
+- Redirect After Order
+- Redirect Before Order
+
+All payment methods available use one of those workflows.
+
+## Direct Order
+
+_Examples:_ check, wire payments, free…
+
+When Customers select the payment method and finish the checkout process, their order is placed immediately without any extra payment step involved and they are redirected on the checkout success page.
+
+It is sometimes possible to display additional input (or external payment forms) during the payment method selection step in Front-Commerce so that Customers can Authorize the payment. When submitting the order, the Direct Order workflow could include additional data (such as an authorization payment token) that could be captured synchronously.
```

Comment:
Is it better to talk about a token here or should we talk about things like customer's bank information for wire payments, etc. Not sure that any payment method would capture synchronously a payment.